lib/puppet/provider/vc_vsan_health_performance/default.rb

```
@@ -9,6 +9,12 @@
   def create
     Puppet.debug("Configuring VSAN performance service")
     vsan.vsanPerformanceManager.VsanPerfCreateStatsObjectTask(:cluster => cluster).onConnection(vim).wait_for_completion
+  rescue => ex
+    if ex.message =~ /FileAlreadyExists/
+      Puppet.debug("FileAlreadyExists error message received. Health services is aleady enabled. Need to ignore this error")
+    else
+      raise ex.message
```

Here we may be hiding other exception details, just use `raise` to re-raise the existing exception.